packages/player/.env.production

```diff
@@ -0,0 +1,2 @@
+# Automatically generated from package.json
```

Comment:
Should this file be in `git` or not?

Reply:
Should be safe to gitignore I guess. As long as we make sure to run the script before Vite. I'll update.